dist/demo.LICENSES.txt

```diff
@@ -1,4 +1,4 @@
-/** @license React v16.13.1
+/** @license React v16.13.0
```

Comment:
the React version in the licence file here & below seems to have reverted to an older version on the PR branch. 🤔